src/Network/NWTcpMetadata.cs

```diff
@@ -0,0 +1,25 @@
+//
+// NWTcpMetadata.cs: Bindings the Netowrk nw_protocol_metadata_t API that is an Tcp.
+//
+// Authors:
+//   Manuel de la Pena <mandel@microsoft.com>
+//
+// Copyrigh 2019 Microsoft
+//
+using System;
+using ObjCRuntime;
+using Foundation;
+using CoreFoundation;
+
+namespace Network {
+
+	[TV (12,0), Mac (10,14), iOS (12,0), Watch (6,0)]
+	public class NWTcpMetadata : NWProtocolMetadata {
+
+		internal NWTcpMetadata (IntPtr handle, bool owns) : base (handle, owns) {}
+
+		public uint AvailableReceiveBuffer () => nw_tcp_get_available_receive_buffer (GetCheckedHandle ());
```

Comment:
why not a property ?

The `=>` syntax is easy to mis-read between methods and properties. However I don't see a good reason for not using a property.